boards/riscv/tl321x/tl3218x-common.dtsi

```diff
@@ -104,11 +104,11 @@
 };
 
 &ram_ilm {
-	reg = <0x00068000 0x00018000>;
+	reg = <0x00068000 0x00020000>;
 };
 
 &ram_dlm {
-	reg = <0x00080000 0x00008000>;
+	reg = <0x00088000 0x00000001>;
```

Comment:
Here base memory address and size is provided.
If this chip doesn't have DLM area just remove its record.
I don't think that chip has 1 byte.

Reply:
If chip has only one momory area it's enough just set this area in .dts chosen
And standard Zephyr linker will handle it...

Reply:
yes , i know what you mean, but acutally i try a lot way , only by this can get the compile result which seems normally 
![compile_good](https://github.com/user-attachments/assets/7897d9d9-b3eb-4e12-9c63-b7f7670d13e1)
      if i delete the dlm section and adjust the code which can compile suc , the compile result will be abnormal, and hard to understand .
![compile_not_good](https://github.com/user-attachments/assets/34c73ab1-af88-42d0-bb26-d727720d8401)

Reply:
I agree here with Andriy, we can't use such workaround.
There should be other clean solutions.

Reply:
@andriy-bilynskyy  need your help to find a way to clear the RAM in the Memory region .

Reply:
@s07641069  I had a quick quick talk with Haiwen.  He has NO more creative way to fix the issue mentioned.  In my opinion, we will have to accept this workaround as a temporary solution.

Reply:
copy andiriy' link https://github.com/telink-semi/zephyr/commit/b2340181f4611d8cfd1e1b6cc99d5b52e49f6224